docs/source/scientific_papers.rst

```diff
@@ -0,0 +1,85 @@
+.. _scientific-papers:
+
+=================
+Scientific Papers
+=================
+
+The **signac** framework has been cited over 50 times, and has been used in a range of research fields with various types of computational workflows.
+Here, we showcase a few examples of scientific papers that have used **signac**.
+To find additional papers using **signac**, please refer to citations of the articles mentioned in :ref:`acknowledge` or this list of `citing works on Google Scholar <https://scholar.google.com/scholar?cites=14270565839129827405,11696507121891896813>`__.
+
+We encourage users to add publications citing **signac** to this list via a pull request.
+To add a paper, put citation information in ``signac.bib`` and follow the formatting of this page.
```

Comment:
```suggestion
To add a paper, put citation information in `signac.bib <https://github.com/glotzerlab/signac-docs/edit/master/docs/source/signac.bib>`__ and follow the formatting of this page.
```
I've actually had a little bit of trouble finding that file. So this might make it easier.